main.go

```diff
@@ -109,8 +124,14 @@ func start() {
 }
 
 func report() {
+	// Calculate the actual output size.
```

Comment:
Can we covert this into a function?

Then, we can use it at line 101 as well.

Reply:
Done